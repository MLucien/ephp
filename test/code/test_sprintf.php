<?php

echo sprintf("%%h (no existent) = %h\n");
echo sprintf("%%'.-20s = %'.-20s\n", "hello");
echo sprintf("%%'.20s  = %'.20s\n", "hello");
echo sprintf("%%2s     = %2s\n", "hello");

