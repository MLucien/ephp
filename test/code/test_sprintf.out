
Warning: sprintf(): Too few arguments in {{CWD}}/test/code/test_sprintf.php on line 3
%'.-20s = hello...............
%'.20s  = ...............hello
%2s     = hello
